hooks/tk-photoshopcc/render_media.py

```diff
@@ -15,6 +15,40 @@
 
 
 class RenderMedia(HookBaseClass):
+    def pre_render(
+        self,
+        input_path,
+        output_path,
+        width,
+        height,
+        first_frame,
+        last_frame,
+        version,
+        name,
+        color_space,
+    ):
+        """
+        Callback executed before the media rendering
+
+        :param path:            Path to the input frames for the movie
+        :param output_path:     Path to the output movie that will be rendered
+        :param width:           Width of the output movie
+        :param height:          Height of the output movie
+        :param first_frame:     The first frame of the sequence of frames.
+        :param last_frame:      The last frame of the sequence of frames.
+        :param version:         Version number to use for the output movie slate and burn-in
+        :param name:            Name to use in the slate for the output movie
+        :param color_space:     Colorspace of the input frames
+
+        :returns:               Location of the rendered media
+        :rtype:                 str
+        """
+
+        engine = sgtk.platform.current_engine()
+
+        if not engine.adobe.get_active_document():
+            raise RuntimeError("No active document found.")
```

Comment:
The photoshop integration crash if there's no active document when trying to save something as JPEG